shell/app/modules/project/pages/test-manage/case/case-drawer/index.tsx

```diff
@@ -212,6 +214,9 @@ const CaseDrawer = ({ visible, scope, onClose, afterClose, afterSave, caseList }
   };
   const checkName = (e: React.FocusEvent<HTMLInputElement>) => {
     const name = e.target.value;
+    if (name === preData.current.name) {
```

Comment:
I don't get the point of persist case detail as ref
what's the difference of comparing with props.caseDetail directly?